geometric/tests/test_run_json.py

```diff
@@ -0,0 +1,56 @@
+"""
+A set of tests for using the QCEngine project
+"""
+
+import numpy as np
+import json, os, shutil
+from . import addons
+import geometric
+
+@addons.using_qcengine
+@addons.using_rdkit
+def test_run_json():
+    # create a test input json file
+    qc_schema_input = {
+        "schema_name": "UFF gradient",
+        "schema_version": "v0.1",
+        "molecule": {
+            "geometry": [
+                0.0,  0.0,              -0.1294769411935893, 
+                0.0, -1.494187339479985, 1.0274465079245698,
+                0.0,  1.494187339479985, 1.0274465079245698
+            ],
+            "symbols": ["O", "H", "H"],
+            "connectivity": [[0, 1, 1], [0, 2, 1]]
+        },
+        "driver": "gradient",
+        "model": {
+            "method": "UFF",
+            "basis": None
+        },
+        "keywords": {},
+        "program": "rdkit"
+    }
+    in_json_dict = {
+        "schema_name": "qc_schema_optimization_input",
+        "schema_version": 1,
+        "geometric_options": {
```

Comment:
Can we switch this to "keywords" just like in the I/O spec?

Reply:
Done